input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,395 @@
+Compliance refers to a resource meeting all the requirements, constraints, and standards specified in a particular FHIR profile or an implementation guide. These requirements can include mandatory elements, cardinality constraints, data types, terminology bindings, and extensions. A resource may be compliant with multiple implementation guides. For example, in addition to AU Core profiles, a FHIR resource may be compliant with profiles published in project implementation guides, other national implementation guides, or international implementation guides.
 
-A resource may be conformant to multiple implementation guides. In addition to AU Core profiles, a FHIR resource may be conformant to profiles published in project implementation guides, other national implementation guides, or international implementation guides.
+### Comparison from AU Core to International Patient Access, International Patient Summary, and US Core
```

Comment:
First heading is Profile comparison & don't put the IGs in the title

Reply:
changed